### DISCUSSION TITLE
Creating a Foley/Sound Effects track for processing movies

### DISCUSSION THREAD

**Discussion by openartist** [Anjok07/ultimatevocalremovergui#1406] [Ideas]
Has there been RECENT discussion about creating an option to have a third track for sound effects, generally multiple tracks, when processing audio files from video? I understand that there would be some new training that would need to be done. Would be an amazing feature to add. Seems like it's been proposed multiple times but no work has been done.

**Comment by HopeAndFaithStudios** [comment]
Well, there's the BandIt Plus model (which separates music, speech and sound effects into 3 tracks), but alas, it's only on MVSep and not on UVR5 at the moment. I hope they consider adding this model though, it would be SO epic if it was taken into consideration.
